pages/common/mongoexport.md

```diff
@@ -0,0 +1,32 @@
+# mongoexport
+
+> Produce exports of data stored in a MongoDB instance formatted as JSON or CSV.
+> More information: <https://docs.mongodb.com/database-tools/mongoexport/>.
+
+- Export documents in JSON format to stdout:
+
+`mongoexport --uri={{mongodb_uri}} --collection={{collection_name}}`
```

Comment:
What does `--uri` do and why is it only in this example?

Reply:
it's basically a connection string.

Reply:
e.g. `mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass&ssl=false`

But @navarroaxel why is it only in this example? 🤔

Reply:
because there are a lot of parameters and I tried to reduce them.

Reply:
Is there any official docs or language we can use as a reference here?

Reply:
https://docs.mongodb.com/manual/reference/connection-string/#connection-string-uri-format

Reply:
Sounds like the official docs use connection string here